boards/arm/ip_k66f/support/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
We should probably add a copyright/license on this file.  I know most of the others don't have it.

Reply:
@galak Added the SPDX-License-Identifier. Thanks.